doc/reference/systems/interactions/potentials.rst

```diff
@@ -186,3 +186,25 @@ units:
     [[pairs]]
     atoms = ["A", "B"]
     morse = {depth = "40 kJ/mol", a = "2.0 rad^-1", x0 = "109.7 deg"}
+
+
+Gaussian potential
+------------------
+
+This potential is usually used to describe energy wells and is expressed as:
+
+.. math::
+
+    V(r) = -a \exp(-b r^2)
```

Comment:
What do you think of using another definition, namely `V = - depth exp(- (r / sigma)^2)` ? It feels clearer to me. We could also use epsilon instead of depth, to be closer to the LJ definition.

Reply:
I went with what I found in papers that use this potential and the [docs of LAMMPS](http://lammps.sandia.gov/doc/pair_gauss.html). We can change it though if you find other expressions more suitable.

Reply:
If they are previous occurrence of the potential with this form, it's fine for me =)